### PR TITLE
RHBRMS-75: Increase code coverage

### PR DESCRIPTION
**Pull request by jomarko**
As result of moving as much as possible internal QE selenium tests into community this PR increases code coverage for upload of xls decision table that has not valid content.

### CODE REVIEW

**Comment by manstis** [comment]
@jomarko Please let me know when this is ready for another look... test failing at the moment.

**Comment by jomarko** [comment]
@manstis sure, I will fix it and let you know.

**Comment by jomarko** [comment]
@manstis it should pass now. Feel free to have a look and let me know if you find something more to fix.

File: drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-client/src/test/java/org/drools/workbench/screens/dtablexls/client/editor/DecisionTableXLSEditorViewImplTest.java [hunk 1]
```diff
@@ -16,19 +16,44 @@
...
+import static org.mockito.Mockito.verify;
 
 @RunWith(GwtMockitoTestRunner.class)
 public class DecisionTableXLSEditorViewImplTest {
 
+    private static final String SERVLET_URL = "dtablexls/file?clientId=123";
     private DecisionTableXLSEditorViewImpl view;
 
+    @Mock
+    AttachmentFileWidget attachemntFileWidget;
```
Comment (manstis) [inline drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-client/src/test/java/org/drools/workbench/screens/dtablexls/client/editor/DecisionTableXLSEditorViewImplTest.java]: Typo "attach**em**nt.."

**Comment by jomarko** [comment]
@manstis sure. No problem. It should be fine now.

**Comment by manstis** [comment]
Thanks @jomarko

**Pull request merged**